attest/core/data/test/quote_ok_str.txt

```diff
@@ -1 +1 @@
9, 11, 137, 214, 67, 42, 33, 171, 42, 220, 253, 190, 174, 184, 22, 189, 183, 169, 136, 166, 123, 80, 44, 146, 231, 180, 166, 131, 184, 141, 246, 255, 198, 73, 210, 78, 117, 214, 29, 191, 207, 138, 160, 158, 34, 218, 95, 184, 75, 36, 250, 25, 8, 248, 211, 56, 39, 123, 191, 141, 195, 198, 224, 96, 172, 229, 81, 183, 234, 242, 211, 253, 187, 252, 56, 144, 93, 14, 217, 212, 205, 40, 101, 127, 7, 206, 77, 91, 169, 228, 5, 165, 151, 197, 52, 72, 75, 78, 56, 197, 77, 95, 154, 123, 204, 96, 35, 115, 249, 106, 41, 98, 197, 27, 96, 22, 43, 40, 195, 13, 138, 55, 77, 18, 243, 244, 110, 197, 151, 6, 153, 195, 158, 142, 122, 53, 39, 68, 60, 101, 205, 86, 171, 240, 173, 17, 83, 66, 110, 11, 163, 46, 223, 132, 7, 15, 43, 250, 58, 50, 167, 70, 59, 205, 174, 192, 196, 108, 233, 51, 120, 153, 193, 57, 158, 104, 176, 78, 211, 100, 9, 196, 155, 221, 210, 219, 43, 252, 64, 255, 122, 27, 40, 30, 44, 79, 169, 96, 124, 103, 143, 250, 79, 122, 125, 147, 207, 212, 97, 0, 71, 238, 182, 39, 85, 134, 18, 64, 238, 210, 115, 21, 140, 51, 221, 23, 245, 57, 210, 72, 161, 179, 161, 151, 217, 134, 249, 197, 45, 67, 162, 70, 6, 10, 106, 59, 255, 33, 233, 113, 216, 47, 27, 254, 198, 84, 179, 18, 101, 240, 233, 128, 165, 74, 41, 226, 186, 138, 202, 107, 255, 226, 118, 104, 221, 33, 162, 104, 1, 0, 0, 212, 106, 13, 193, 255, 196, 152, 107, 153, 75, 11, 110, 253, 97, 6, 55, 215, 178, 43, 242, 24, 136, 198, 41, 106, 154, 83, 140, 173, 184, 172, 93, 143, 65, 86, 172, 1, 219, 23, 87, 155, 144, 172, 212, 225, 48, 245, 134, 83, 115, 168, 77, 46, 161, 49, 238, 79, 50, 109, 22, 157, 65, 209, 181, 82, 218, 170, 76, 81, 76, 138, 225, 36, 81, 133, 188, 207, 79, 76, 155, 107, 95, 62, 60, 236, 147, 33, 108, 209, 174, 2, 63, 3, 21, 95, 191, 5, 255, 7, 1, 183, 250, 221, 41, 161, 240, 151, 49, 139, 155, 162, 164, 22, 232, 109, 49, 49, 133, 156, 122, 234, 69, 197, 140, 218, 74, 90, 111, 168, 51, 234, 188, 240, 165, 32, 215, 52, 45, 81, 77, 189, 92, 159, 168, 94, 216, 157, 142, 133, 52, 42, 59, 111, 27, 193, 20, 76, 17, 16, 44, 214, 64, 158, 158, 147, 238, 103, 46, 191, 254, 217, 177, 123, 14, 118, 106, 149, 86, 255, 11, 192, 218, 91, 170, 201, 29, 227, 157, 29, 234, 204, 53, 228, 3, 103, 149, 189, 94, 191, 231, 194, 232, 57, 41, 33, 117, 127, 35, 85, 253, 181, 255, 12, 61, 104, 133, 113, 44, 243, 153, 208, 89, 138, 149, 223, 54, 234, 52, 241, 15, 62, 73, 6, 4, 61, 145, 52, 73, 210, 78, 18, 34, 133, 131, 208, 113, 58, 237, 138, 84, 197, 48, 205, 196, 194, 138, 207, 201, 22, 254, 123, 169, 107, 231, 0, 237, 46, 204, 179, 186, 104, 116, 161, 186, 221, 42, 30, 248, 11, 172, 145, 105, 48, 86, 102, 179, 48, 145, 248, 173, 147, 94, 49, 211, 237, 51, 37, 22, 66, 120, 123, 22, 30, 3, 34, 216, 145, 227, 161, 250, 23, 18, 81, 120, 59, 97, 179, 179, 62, 93, 255, 171, 169, 1, 14, 227, 62, 68, 88, 64, 224, 228, 147, 121, 157, 114, 129, 166, 129, 216, 107, 26, 253, 5, 207, 217, 11, 46, 175, 5, 132, 188, 207, 126, 239, 229, 217, 116, 122, 46, 82, 160, 229, 199, 160, 165, 210, 255, 31, 26, 245, 144, 215, 6, 74, 189]) }
```

Comment:
In the original version, the report body was wrapped in a `Result`, this unwraps it.